src/cattr/_compat.py

```diff
@@ -161,21 +161,11 @@ def is_mapping(type):
             and issubclass(type.__origin__, TypingMapping)
         )
 
-    bare_list_args = List.__args__
-    bare_seq_args = TypingSequence.__args__
-    bare_mapping_args = TypingMapping.__args__
-    bare_dict_args = Dict.__args__
-    bare_mutable_seq_args = TypingMutableSequence.__args__
-
     def is_bare(type):
-        args = type.__args__
-        return (
-            args == bare_list_args
-            or args == bare_seq_args
-            or args == bare_mapping_args
-            or args == bare_dict_args
-            or args == bare_mutable_seq_args
-        )
+        # Lower-cased generics in 3.7-8 do not have `__args__` attribute.
+        # Generics imported from `typing` module which do not have their
```

Comment:
This is incorrect and now, although the reported bug is fixed, everything else is broken lol.

Sorry, brain fart, will fix it soon.